### PR TITLE
Fix details

### PR DESCRIPTION
**Pull request by elboulangero**
A few fixes I made while getting started with debos. Just details..

### CODE REVIEW

**Comment by elboulangero** [comment]
Thanks for the link, I gave it a try and ended up with simpler commit messages, hope you like it ;)

**Comment by elboulangero** [comment]
Any news on that? Is there anything else you want to change?

**Comment by sjoerdsimons** [comment]
Did your ocmmit messages actually change?

**Comment by elboulangero** [comment]
Yes, I don't know if there's a way to see that in GitLab though.

**Comment by sjoerdsimons** [comment]
Well even https://github.com/elboulangero/debos/tree/fix-details still has commit messages with only a subject so...

**Comment by elboulangero** [comment]
OK I didn't really understand that it was about more detailed commit messages. I pushed again a revised version. Thanks.

File: filesystem.go [hunk 1]
```diff
@@ -63,7 +69,10 @@ func CopyTree(sourcetree, desttree string) error {
 		target := path.Join(desttree, suffix)
 		switch info.Mode() & os.ModeType {
 		case 0:
-			CopyFile(p, target, info.Mode())
+			err := CopyFile(p, target, info.Mode())
+			if err != nil {
+				log.Panicf("Failed to copy file %s: %v", p, err)
```
Comment (sjoerdsimons) [inline filesystem.go]: This should return err instead of panicking; potentially extending the error message to make it more clear what when wrong

**Pull request merged**